packages/web-app-user-management/tests/unit/views/Users.spec.ts

```diff
@@ -21,10 +22,13 @@ const defaultGraphMock = () => {
   }
 
   const graph = mockDeep<Graph>()
-  graph.users.listUsers.mockImplementation(() => mockAxiosResolve({ value: [defaultUser] }))
-  graph.users.getUser.mockImplementation(() => mockAxiosResolve(defaultUser))
+  // graph.users.listUsers.mockImplementation(() => mockAxiosResolve({ value: [defaultUser] }))
+  // graph.users.getUser.mockImplementation(() => mockAxiosResolve(defaultUser))
+  // graph.groups.listGroups.mockImplementation(() => mockAxiosResolve({ value: [] }))
 
-  graph.groups.listGroups.mockImplementation(() => mockAxiosResolve({ value: [] }))
+  graph.users.listUsers.mockResolvedValue({ data: { value: [defaultUser] } } as AxiosResponse)
+  graph.users.getUser.mockResolvedValue({ data: defaultUser } as AxiosResponse)
+  graph.groups.listGroups.mockResolvedValue({ data: { value: [] } } as AxiosResponse)
```

Comment:
Let's not typecast here, `satisfies` would be ok or we add a helper that accepts a proper interface and returns an AxiosResponse not wrapped in a Promise

And especially let's get rid of the commented out code before merge